core-bundle/src/Resources/public/core.js

```diff
@@ -1724,6 +1748,7 @@ var Backend =
 								ntr = new Element('tr');
 								childs = tr.getChildren();
 								for (i=0; i<childs.length; i++) {
+									Backend.retrieveInteractiveHelp(childs[i].getElements('button,a'));
```

Comment:
Shouldn't this be `'a[title]', 'input[title]', 'button[title]', 'time[title]'` like in line 1323?

Reply:
if then `'a', 'input', 'button', 'time'`, because we want to retrieve the missing title! but in the widget context there are only `button,a` as actions in the markup!